integration/volume/volume_test.go

```diff
@@ -234,3 +236,42 @@ func TestVolumePruneAnonymous(t *testing.T) {
 	assert.Check(t, cmp.Contains(pruneReport.VolumesDeleted, v.Name))
 	assert.Check(t, cmp.Contains(pruneReport.VolumesDeleted, vNamed.Name))
 }
+
+func TestVolumePruneAnonFromImage(t *testing.T) {
+	defer setupTest(t)()
+	client := testEnv.APIClient()
+
+	volDest := "/foo"
+	if testEnv.OSType == "windows" {
+		volDest = "c:/foo"
+	}
+
+	dockerfile := `FROM busybox
+VOLUME ` + volDest
+
+	ctx := context.Background()
+	img := build.Do(ctx, t, client, fakecontext.New(t, "", fakecontext.WithDockerfile(dockerfile)))
+
+	id := container.Create(ctx, t, client, container.WithImage(img))
+	defer client.ContainerRemove(ctx, id, types.ContainerRemoveOptions{})
+
+	inspect, err := client.ContainerInspect(ctx, id)
+	assert.NilError(t, err)
+
+	var volumeName string
+	for _, m := range inspect.Mounts {
+		if m.Destination != volDest {
```

Comment:
Looks like we're back to this part not finding the volume 😞 (I suspect here it expects it to be `c:\foo`)